Automattic/jetpack
# JS Components: Improve donut meter accessibility

Partially address #26049.

#### Changes proposed in this Pull Request:


Some a11y tweaks for the Donut Meter component. Instead of using a hidden TABLE element to describe the meter, we set the role to IMG, add TITLE and DESC elements, and source those via the caller (as props). In this way, the SVG element is directly accessible to screen readers and should be treated correctly.

Removes the need for the TABLE element and any localization as that would be handled by the caller.

#### Other information:


#### Jetpack product discussion



Some discussion in #26049.

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Run `cd projects/js-packages/storybook && npm run storybook:dev` and confirm that you see DonutMeter component in Storybook. It should be under JS Packages → Components → DonutMeter.
2. Choose the Default story and confirm we have zero violations under the Accessibility tab.
3. Open the Storybook page in Firefox and use the Accessibility inspector to confirm no violations/suggestions there.

![Report](https://user-images.githubusercontent.com/40267301/189291263-eeaccaec-0afe-4030-a53c-8c4c3d8cae48.png)
